ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/metastore/HiveMetaStoreAuthorizableEvent.java

```diff
@@ -71,4 +72,10 @@ protected HivePrivilegeObject getHivePrivilegeObjectLocalUri(String uri) {
     return new HivePrivilegeObject(HivePrivilegeObject.HivePrivilegeObjectType.LOCAL_URI, null, uri);
   }
 
+  protected HivePrivilegeObject getHivePrivilegeObject(DataConnector connector) {
+    return new HivePrivilegeObject(HivePrivilegeObject.HivePrivilegeObjectType.DATACONNECTOR, null,
+        connector.getName(), null, null, HivePrivilegeObject.HivePrivObjectActionType.OTHER, null, null,
```

Comment:
We might need connector type and connector url in authorization service right? Can we include them here instead of passing null?

Reply:
`connector url`  and  `connector type` can not be added  as a param here. At present, we think all alter operations have  same permission check logical, and no need to  specially check` connector url` or `connector type`.

Also, there is no related  privilege types of  `connector url`  and  `connector type` in `HivePrivilegeObject `class:
https://github.com/apache/hive/blob/18501dd2e6f0c68038600b4dc69b7197a7827cbd/ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/HivePrivilegeObject.java#L108-L116

Reply:
Ack